LAPACKE `strict-prototypes` compiler warning for `LAPACKE_get_nancheck` prototype

https://github.com/Reference-LAPACK/lapack/blob/8e1e16c6300861f096744a5ab75c3572c4b9b756/LAPACKE/include/lapacke.h#L12578

If compiler warning `-Werror=strict-prototypes` is set, the following message appears:

    include/lapacke.h:12578:1: error: function declaration isn’t a prototype [-Werror=strict-prototypes]

A simple fix: 

`int LAPACKE_get_nancheck(void);`

This issue was originally raised in OpenBLAS: https://github.com/xianyi/OpenBLAS/issues/2619

--- a/LAPACKE/include/lapacke.h
+++ b/LAPACKE/include/lapacke.h
@@ -12575,7 +12575,7 @@ lapack_int LAPACKE_zhetrs_aa_2stage_work( int matrix_layout, char uplo, lapack_i
                    
 /* APIs for set/get nancheck flags */
 void LAPACKE_set_nancheck( int flag );
-int LAPACKE_get_nancheck( );
+int LAPACKE_get_nancheck( void );
 
 #ifdef __cplusplus
 }

